q2_dada2/assets/run_dada_single.R

```diff
@@ -119,12 +126,13 @@ maxLen <- as.numeric(args[[9]]) # Allows Inf
 poolMethod <- args[[10]]
 chimeraMethod <- args[[11]]
 minParentFold <- as.numeric(args[[12]])
-nthreads <- as.integer(args[[13]])
-nreads.learn <- as.integer(args[[14]])
+allowOneOff <- as.logical(args[[13]])
+nthreads <- as.integer(args[[14]])
+nreads.learn <- as.integer(args[[15]])
 # The following args are not directly exposed to end users in q2-dada2,
 # but rather indirectly, via the methods `denoise-single` and `denoise-pyro`.
-HOMOPOLYMER_GAP_PENALTY <- if (args[[15]]=='NULL') NULL else as.integer(args[[15]])
-BAND_SIZE <- as.integer(args[[16]])
+HOMOPOLYMER_GAP_PENALTY <- if (args[[16]]=='NULL') NULL else as.integer(args[[15]])
```

Comment:
```suggestion
HOMOPOLYMER_GAP_PENALTY <- if (args[[16]]=='NULL') NULL else as.integer(args[[16]])
```

Reply:
@leasi looks like you just missed adjusting this index - with this change, all existing tests within q2-dada2 pass! 🥳